templates/content-certificate.php

```diff
@@ -13,7 +13,11 @@
 $cert = new LLMS_User_Certificate( get_the_ID() );
 $uid  = get_current_user_id();
 
-if ( $uid != $cert->get_user_id() && ! llms_can_user_bypass_restrictions( $uid ) ) {
+$is_owner_or_admin 		= ($uid == $cert->get_user_id() || llms_can_user_bypass_restrictions( $uid ));
+$is_shareable					= (get_post_meta( get_the_ID(), '_llms_allow_sharing', true ) === 'yes');
+$is_allow_to_display	= $is_owner_or_admin || $is_shareable;
+
+if ( ! $is_allow_to_display ) {
```

Comment:
Assuming class methods are added (as noted earlier in review):

```suggestion
if ( ! $cert->can_user_view() ) {
```